src/components/FiatInput/useFontSizeInPx.ts

```diff
@@ -0,0 +1,30 @@
+import { RefObject, useEffect, useState } from 'react'
+
+export const useFontSizeInPx = (
+  initialFontSizeAsPropertyValue: string,
+  ref: RefObject<HTMLElement>
+) => {
+  const [textSizeInPx, setTextSizeInPx] = useState<number | null>(null)
+
+  // Get font size rendered by browser that corresponds to initial (possibly themed) value, like 'xl'
+  useEffect(() => {
+    if (!ref.current) {
+      return
+    }
+    const fontSizeWithUnits = getComputedStyle(ref.current).getPropertyValue('font-size')
+    const match = fontSizeWithUnits.match(/\d+/)
+    setTextSizeInPx(parseInt(match![0]))
```

Comment:
Why not just use `calc()` CSS function? this seems heavy handed and I feel there must be a way to accomplish this with just CSS.

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/clamp()

Reply:
Thanks for pointing `clamp()` my way. I think some amount of calculation is also needed to arrive at the font sizes in the arguments of `clamp()`, specifically as the 3rd/max font size. I just read [Font scaling based on width of container on Stack Overflow](https://stackoverflow.com/questions/16056591/font-scaling-based-on-width-of-container) and [Fitting Text to a Container on CSS-Tricks](https://css-tricks.com/fitting-text-to-a-container/). I quite like the svg approach, which doesn't need any JavaScript/calculation.

Reply:
I realised SVG approach won't work, as this an `<Input>` that accepts user input.